frontend/packages/llama-stack-modular-ui/frontend/src/odh/extensions.ts

```diff
@@ -0,0 +1,52 @@
+import type {
+  NavExtension,
+  RouteExtension,
+  AreaExtension,
+} from '@odh-dashboard/plugin-core/extension-points';
+
+const PLUGIN_LLAMA_STACK = 'llama-stack-plugin';
+
+const extensions: (NavExtension | RouteExtension | AreaExtension)[] = [
+  {
+    type: 'app.area',
+    properties: {
+      id: PLUGIN_LLAMA_STACK,
+      devFlags: ['LLama Stack Plugin'],
```

Comment:
Do we need to add `reliantAreas: [SupportedArea.LLAMA_STACK_CHAT_BOT],` here?

Reply:
discussed: should not relying on old featureFlag